packages/renderer/src/lib/preferences/item-formats/FileItem.spec.ts

```diff
@@ -0,0 +1,26 @@
+import '@testing-library/jest-dom/vitest';
+import { test, expect, vi, beforeAll } from 'vitest';
+import { render, screen } from '@testing-library/svelte';
+import type { IConfigurationPropertyRecordedSchema } from '../../../../../main/src/plugin/configuration-registry';
+import FileItem from '/@/lib/preferences/item-formats/FileItem.svelte';
+
+beforeAll(() => {
+  (window as any).getConfigurationValue = vi.fn();
+});
+
+test('Ensure HTMLInputElement', async () => {
+  const record: IConfigurationPropertyRecordedSchema = {
+    id: 'record',
+    title: 'record',
+    parentId: 'parent.record',
+    description: 'record-description',
+    type: 'string',
+    format: 'file',
+  };
+
+  render(FileItem, { record, value: undefined });
+  const input = screen.getByLabelText('record-description');
+  expect(input).toBeInTheDocument();
+
```

Comment:
it looks like we're missing to test the click on the element

Reply:
done